Add word-break property to screen reader only classes.

peterwilsoncc opened the pull request:
Adds `word-break: normal !important` to various screen reader only CSS.

This prevents screen readers from announcing the text as it would appear visually, eg "A P P E A R space V I S U A L L Y".

Trac ticket: https://core.trac.wordpress.org/ticket/64375

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

peterwilsoncc committed:
Add word-break property to screen reader only classes.

Commit:
Automation: Updating built files with changes.

Commit:
Automation: Updating built files with changes.

Commit:
Automation: Updating built files with changes.

Commit:
Automation: Updating built files with changes.

peterwilsoncc committed:
Sentence case for the comments.

Commit:
Automation: Updating built files with changes.

mukeshpanchal27 commented on src/wp-includes/deprecated.php:
```diff
@@ -6176,7 +6176,9 @@ function the_block_template_skip_link() {
 			padding: 0;
 			position: absolute !important;
 			width: 1px;
+			/* Many screen reader and browser combinations announce broken words as they would appear visually. */
```
Are we still needs to update the deprecated codes?

peterwilsoncc replied:
@mukeshpanchal27 sometimes they are sometimes not. I've removed the change in this PR.

joedolson replied:
I put this back in, as I think that it makes sense to address this. Deprecated code can still be used, and would still benefit from this change.

peterwilsoncc committed:
Revert changes in deprecated function.

joedolson committed:
Merge branch 'trunk' into fix/64375-screen-reader-word-break

Commit:
Automation: Updating built files with changes.

joedolson committed:
Merge branch 'trunk' into fix/64375-screen-reader-word-break

Commit:
Automation: Updating built files with changes.

joedolson committed:
Merge branch 'trunk' into fix/64375-screen-reader-word-break

joedolson committed:
Update screen reader text class in three additional cases

One of these is in a deprecated function, but for completeness, I added that as well.

joedolson commented:
Added two additional cases that were omitted in the PR; restored one that had been removed in a deprecated function. Could go either way on that, but I think it's better to fix the issue even if it's going to be rarely used.

joedolson committed:
I screwed up merging conflicts; restore.

joedolson committed:
Update deprecated.php

joedolson committed:
Brain.
